src/Directory.Packages.props

```diff
@@ -4,20 +4,20 @@
   </PropertyGroup>
   <ItemGroup>
     <!--    add your PackageVersion nodes here -->
-    <PackageVersion Include="Microsoft.Azure.Kusto.Ingest" Version="12.1.0" />
-    <PackageVersion Include="serilog" Version="2.12.0" />
-    <PackageVersion Include="Serilog.Formatting.Compact" Version="1.1.0" />
-    <PackageVersion Include="Serilog.Formatting.Compact.Reader" Version="2.0.0" />
-    <PackageVersion Include="Serilog.Sinks.PeriodicBatching" Version="3.1.0" />
```

Comment:
kcsb.SetConnectorDetails(AppName, ClientVersion, "Serilog", "2.12.0");

Change version to 4.0.0